l2tdevtools/build_helpers/wheel.py

```diff
@@ -0,0 +1,236 @@
+    """Moves the wheel from the dist sub directory into the build directory.
+
+    Args:
+      source_helper_object (SourceHelper): source helper.
+
+    Returns:
+      bool: True if the move was successful, False otherwise.
+    """
+    project_name, _ = self._GetWheelFilenameProjectInformation(
+        source_helper_object)
+
+    source_directory = source_helper_object.GetSourceDirectoryPath()
+    if not source_directory:
+      logging.info('Missing source directory of: {0:s}'.format(
+          source_helper_object.project_name))
+      return False
+
+    filenames_glob = os.path.join(
+        source_directory, 'dist', '{0:s}-*-*-*-*.whl'.format(project_name))
+    filenames = glob.glob(filenames_glob)
+
+    if len(filenames) != 1:
+      logging.error('Unable to find wheel file: {0:s}.'.format(filenames_glob))
+      return False
+
+    _, _, wheel_filename = filenames[0].rpartition(os.path.sep)
+    if os.path.exists(wheel_filename):
+      logging.warning('Wheel file already exists.')
+    else:
+      logging.info('Moving: {0:s}'.format(filenames[0]))
+      shutil.move(filenames[0], '.')
+
+    return True
+
+  def CheckBuildDependencies(self):
+    """Checks if the build dependencies are met.
+
+    Returns:
+      list[str]: build dependency names that are not met or an empty list.
+    """
+    missing_packages = []
+    for package_name in self._project_definition.build_dependencies:
+      if package_name not in ('fuse', 'libcrypto', 'zlib'):
+        missing_packages.append(package_name)
+
+    return missing_packages
+
+  def CheckBuildRequired(self, source_helper_object):
+    """Checks if a build is required.
+
+    Args:
+      source_helper_object (SourceHelper): source helper.
+
+    Returns:
+      bool: True if a build is required, False otherwise.
+    """
+    project_name, project_version = self._GetWheelFilenameProjectInformation(
+        source_helper_object)
+
+    wheel_filename = '{0:s}-{1:s}-*-*-*.whl'.format(
+        project_name, project_version)
+
+    return not os.path.exists(wheel_filename)
+
+  def Clean(self, source_helper_object):
+    """Cleans the build and dist directory.
+
+    Args:
+      source_helper_object (SourceHelper): source helper.
+    """
+    # Remove previous versions of wheels.
+    project_name, project_version = self._GetWheelFilenameProjectInformation(
+        source_helper_object)
+
+    filenames_to_ignore = '{0:s}-{1:s}-.*-.*-.*.whl'.format(
+        project_name, project_version)
+    filenames_to_ignore = re.compile(filenames_to_ignore)
+
+    filenames_glob = '{0:s}-*-*-*-*.whl'.format(project_name)
+    filenames = glob.glob(filenames_glob)
+
+    for filename in filenames:
+      if not filenames_to_ignore.match(filename):
+        logging.info('Removing: {0:s}'.format(filename))
+        os.remove(filename)
+
+
+class ConfigureMakeWheelBuildHelper(WheelBuildHelper):
+  """Helper to build Python wheel packages (.whl)."""
```

Comment:
Please update docstring to indicate what this class does, and how it's from the other classes with the same docstring.

Reply:
added additional text